retrofit/src/main/java/retrofit2/KotlinExtensions.kt

```diff
@@ -14,9 +14,51 @@
  * limitations under the License.
  */
 
-// Hide the class from Java consumers.
-@file:JvmName("-KotlinExtensions")
+@file:JvmName("KotlinExtensions")
 
 package retrofit2
 
+import kotlinx.coroutines.suspendCancellableCoroutine
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+
 inline fun <reified T> Retrofit.create(): T = create(T::class.java)
+
+suspend fun <T : Any> Call<T>.await(): T {
```

Comment:
This is neat.

Reply:
And public API for people to use if they already have `Call`s! Makes it a win-win really.